dataland-community-manager/src/main/kotlin/org/dataland/datalandcommunitymanager/api/RequestApi.kt

```diff
@@ -91,5 +95,100 @@ interface RequestApi {
     fun getAggregatedDataRequests(
         @RequestParam identifierValue: String? = null,
         @RequestParam dataTypes: Set<DataTypeEnum>? = null,
-    ): List<AggregatedDataRequest>
+        @RequestParam reportingPeriod: String? = null,
+    ): ResponseEntity<List<AggregatedDataRequest>>
+
+    /**
+     * A method to post a single request to Dataland.
+     * @param singleDataRequest includes necessary info for the single request
+     * @return response after posting a single data request to Dataland
+     *
+     */
+    @Operation(
+        summary = "Send a single request",
+        description = "A single of data requests for specific frameworks and companies is being sent.",
+    )
+    @ApiResponses(
+        value = [
+            ApiResponse(responseCode = "200", description = "Successfully processed a single data request."),
+        ],
+    )
+    @PostMapping(
+        value = ["/single"],
+        consumes = ["application/json"],
+        produces = ["application/json"],
+    )
+    @PreAuthorize("hasRole('ROLE_USER')")
+    fun postSingleDataRequest(
+        @Valid @RequestBody
+        singleDataRequest: SingleDataRequest,
+    ):
+        ResponseEntity<List<StoredDataRequest>>
```

Comment:
Should the aggregated request endpoint be extended to filter for the status? So that the user would be able to get all / all open / all closed requests?

Reply:
Good point. We must not forget that the aggregated GET does not return a list of stored data requests, but a different type that does not hold the status. IMO, the first question to answer would be whether resolved requests should also be included in the count. I would say no, i.e. the aggregation only takes open requests into account. This would make a status filter redundant. What is your opinion @emanueldfine ?

Reply:
In the future it would be good if it could filter by the requestStatus. Currently however we won't implement it in this story